tools/defradb.containerfile

```diff
@@ -9,11 +9,16 @@ WORKDIR /repo/
 COPY go.mod go.sum Makefile ./
 RUN make deps:modules
```

Comment:
thought: Maybe we can get by not building lens deps here as they might be only dev dependencies. I will make a ticket and it can be taken care of outside the scope of this PR.
New Issue here:
https://github.com/sourcenetwork/defradb/issues/1707